website/src/data/users.tsx

```diff
@@ -1366,6 +1366,14 @@ const Users: User[] = [
     source: 'https://github.com/redis-developer/redis-developer.github.io',
     tags: ['opensource', 'product', 'design', 'favorite'],
   },
+  {
+    title: 'Reddit Image Fetcher',
+    description: 'A JavaScript package for fetching reddit images, memes, wallpapers and more',
+    preview: require('./showcase/reddit-image-fetcher.png'),
+    website: 'https://arifszn.github.io/reddit-image-fetcher',
+    source: 'https://github.com/arifszn/reddit-image-fetcher',
```

Comment:
This does not seem to contain your website's source?

Reply:
I was following `RactivePlayer`'s structure and  I had the impression the source is the library's source.

Reply:
Hmmm, indeed, that one probably slipped through the review. You do need to make the `source` point to your website's source.

Reply:
I see. Also, there are some more examples where the website source is not presented in the mentioned source URL e.g. `Node SerialPort`, `Nocalhost`. And other repos have the source as well as the library's source. So, you can guess why I misunderstood it as the library's source.

Currently, the website source is in another private repo. If it's the requirement then, I will move the website source to the library's subfolder. Which will take some time so you can close this PR for now. Or, source with `null` value is acceptable like `Paubox`?

Reply:
You can do either: public your website's source, or indicate that your website is not open-source. No need to close the current PR though, we can always wait ;D

Thanks for pointing those sites out, I've noted.

Reply:
Thanks. I will surely make the doc source open source soon and update the link. Could have made the repo public now but not super proud of the commit messages :p. For now, keeping the source as `null`.